### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by Blai01** [NagiosEnterprises/nrpe#121]
I have a problem installing NRPE 3.1.0 in a SUSE Linuxe Enterprise 12 machine.

First I download the nrpe 3.1.0 tar

I decompress it 

I cd into its directory

I run:

./configure --disable-ssl --enable-command-args

Then I run:

make all

Response is:

cd ./src/; make
make[1]: Entering directory '/tmp/nrpe-3.1.0/src'
gcc -g -O2 -DHAVE_CONFIG_H -I ../include -I ./../include -o nrpe ./nrpe.c ./utils.c ./acl.c  -lnsl
/tmp/cckjPq8N.o: In function `logit':
/tmp/nrpe-3.1.0/src/./utils.c:518: undefined reference to `va_start'
/tmp/nrpe-3.1.0/src/./utils.c:534: undefined reference to `va_end'
collect2: error: ld returned 1 exit status
Makefile:48: recipe for target 'nrpe' failed
make[1]: *** [nrpe] Error 1
make[1]: Leaving directory '/tmp/nrpe-3.1.0/src'
Makefile:66: recipe for target 'all' failed
make: *** [all] Error 2

NRPE version 3.0.1 installs fine when typing these commands

**Comment by box293** [comment]
I cannot reproduce your issue. I followed these steps https://support.nagios.com/kb/article.php?id=515#SUSE and downloaded 3.1.0 (https://github.com/NagiosEnterprises/nrpe/archive/nrpe-3.1.0.tar.gz) and `./configure --disable-ssl --enable-command-args` completes fine.

I then ran `sudo zypper update` to be sure I had the latest version of everything. Even after that I could not reproduce your issue.

```
cat /etc/*elease
NAME="SLES"
VERSION="12-SP2"
VERSION_ID="12.2"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP2"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp2"
SUSE Linux Enterprise Server 12 (x86_64)
VERSION = 12
PATCHLEVEL = 2
```
